i18n/tr.yaml

```diff
@@ -0,0 +1,25 @@
+- id: prev_page
+  translation: "Önceki Sayfa"
+
+- id: next_page
+  translation: "Sonraki Sayfa"
+
+- id: read_time
+  translation:
+    one : "1 dk"
+    other: "{{ .Count }} dk"
+
+- id: toc
+  translation: "İçindekiler"
+
+- id: translations
+  translation: "Çeviriler"
+
+- id: home
+  translation: "Ana Sayfa"
+
+- id: code_copy
+  translation: "Kopyala"
+
+- id: code_copied
+  translation: "Kopyalandı !"
```

Comment:
Also the space before `!` should't be here I think ?

Reply:
Updated.